src/main/java/core/basesyntax/service/amount/BalanceHandlerImpl.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.service.amount;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceHandlerImpl implements OperationHandler {
+    @Override
+    public void updateAmountInStorage(FruitTransaction fruitTransaction) {
+        Storage.fruitMap.put(fruitTransaction.getFruitType(),
+                (Storage.fruitMap.get(fruitTransaction.getFruitType())
+                        + fruitTransaction.getAmount()));
```

Comment:
```suggestion
    public void updateAmountInStorage(FruitTransaction fruitTransaction) {
        Storage.fruitMap.put(fruitTransaction.getFruitType(), fruitTransaction.getAmount()));
```
I think balance sets starting amount

Reply:
In case where operation balance isn`t first our value will be reset so I think it be better to update value, no set